tools/download.go

```diff
@@ -319,29 +319,61 @@ func stringInSlice(str string, list []string) bool {
 	return false
 }
 
-func findBaseDir(dirList []string) string {
-	if len(dirList) == 1 {
-		return filepath.Dir(dirList[0]) + "/"
-	}
-	baseDir := ""
-	// https://github.com/backdrop-ops/contrib/issues/55#issuecomment-73814500
-	dontdiff := []string{"pax_global_header"}
```

Comment:
This was needed to take into account some packets created by github releases; maybe it's a good idea to restore it? I'll try to find the file that was provoking this to improve tests

Reply:
solved in https://github.com/arduino/arduino-create-agent/pull/263/commits/25c33b88e1de4e6090e21a027cc6a54186fce321